internal/web/api/k3s.go

```diff
@@ -0,0 +1,66 @@
+package api
+
+import (
+	"github.com/weka/gohomecli/internal/bundle"
+	"github.com/weka/gohomecli/internal/k3s"
+	"net/http"
+)
+
+func isK3sEnabled() bool {
+	return bundle.IsBundled()
+}
+
+type K3sInstallRequest struct {
+	Interface   string   `json:"interface"`
+	Hostname    string   `json:"hostname"`
+	NodeIP      string   `json:"node_ip"`
+	ExternalIPs []string `json:"external_ips"`
+}
+
+func postK3sInstall(w http.ResponseWriter, r *http.Request) {
```

Comment:
got a little bit confustion from naming `postK3sInstall` sounds like hook which runs after k3s installed 😄

Reply:
yeah, this one is a bit tricky. I'll rename POST methods just to an action like 'k3sInstall`